shared/constants/app.ts

```diff
@@ -77,6 +77,13 @@ export const SNAP_DIALOG_TYPES = {
 };
 ///: END:ONLY_INCLUDE_IN
 
+///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
+export const SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES = {
```

Comment:
As mentioned in another comment, we try and maintain all the approval types in the enum inside `@metamask/controller-utils`.

This makes sense for now to avoid duplication but could be removed in a subsequent PR.

Reply:
Made an issue to track this work: https://github.com/MetaMask/accounts-planning/issues/70